### PR TITLE
LPS-140725 Empty page is displayed when adding or editing an Object entry that contains a dot character on a string field

### PR DESCRIPTION
**Pull request by eli-rlima**
Hi reviewers!

This changes are necessary because the `Liferay.Util.submitForm` don't work adequately to our use case. I changed the way to do the form submit from `Liferay.Util.submitForm` to do a fetch in Object API.

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp [hunk 1]
```diff
@@ -99,9 +95,44 @@ portletDisplay.setURLBack(backURL);
 						{}
 					);
 
-					ddmFormValues.value = JSON.stringify(values);
-
-					Liferay.Util.submitForm(form);
+					Liferay.Util.fetch(
+						'/o<%= objectDefinition.getRESTContextPath() %>',
```
Comment (interaminense) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp:99]: `objectDefinition.getRESTContextPath()` returns the path related to current endpoint?

Comment (eli-rlima) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp:99]: Yes, @interaminense!

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by eli-rlima** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by guilhermedcamacho** [comment]
ci:forward

**Pull request closed without merging**